src/Taxjar/Taxjar.csproj

```diff
@@ -31,4 +31,24 @@
     <PackageReference Include="RestSharp" Version="106.10.1" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="3.1.10"/>
+    <PackageReference Include="Bet.Extensions.Options" Version="3.1.10"/>
```

Comment:
It would be preferred to strictly use the `Microsoft.Extensions.Options` package here instead.

Reply:
I can defiantly remove this dependency but fyi this code registers options that will be updated if the underlining configuration provider has changed. This is very helpful in situation where you don't want to restart the application for new options values to be propagated to the app.